apps/arweave/c_src/Makefile

```diff
@@ -84,12 +86,20 @@ RX4096_OBJECTS = $(addsuffix .o, $(basename $(RX4096_SOURCES)))
 RXSQUARED_OBJECTS = $(addsuffix .o, $(basename $(RXSQUARED_SOURCES)))
 VDF_OBJECTS = $(addsuffix .o, $(basename $(VDF_SOURCES)))
 
-UNAME_SYS := $(shell uname -s)
+# NOTE tabs here will cause build fail
 ifeq ($(UNAME_SYS), Linux)
   $(C_SRC_DIR)/vdf/vdf_exp_x86.o: CXXFLAGS += -msha
 endif
 ifeq ($(UNAME_SYS), Darwin)
   $(C_SRC_DIR)/vdf/vdf_exp_arm.o:  CXXFLAGS += -march=armv8-a+crypto
+  $(C_SRC_DIR)/vdf/vdf_exp_arm2.o: CXXFLAGS += -march=armv8-a+crypto
+endif
+ifeq ($(UNAME_SYS), Darwin)
```

Comment:
```suggestion
```

Reply:
ah got it. Okay yeah if the space/tabs are different in each if block makes sense to keep them separate